srv/pillar/ceph/disk_led.sls

```diff
@@ -0,0 +1,15 @@
+# This is the default configuration for the storage enclosure LED blinking.
+# The placeholder {device_file} will be replaced with the device file of
+# the disk when the command is executed.
+#
+# Have a look into the /srv/pillar/ceph/README file to find out how to
+# customize this configuration per minion/host.
+
+disk_led:
+  cmd:
+    ident:
+      'on': lsmcli local-disk-ident-led-on '{device_file}'
```

Comment:
This (and the three other lsmcli invocations) all need `--path` added, e.g.: `lsmcli local-disk-ident-led-on --path '{device_file}'`

Reply:
Done